modules/linear_boltzmann_solvers/discrete_ordinates_solver/sweep/communicators/cbc_async_comm.cc

```diff
@@ -97,9 +95,8 @@ CBC_ASynchronousCommunicator::ReceiveData()
 {
   CALI_CXX_MARK_SCOPE("CBC_ASynchronousCommunicator::ReceiveData");
 
-  typedef std::pair<uint64_t, unsigned int> CellFaceKey; // cell_gid + face_id
-
-  std::map<CellFaceKey, std::vector<double>> received_messages;
+  // Map indexed by a pair (cell_gid, face_id) storing received angular flux data
+  std::map<std::pair<uint64_t, unsigned int>, std::vector<double>> received_messages;
```

Comment:
here's another example where I find the typedef to help readability. `typedef std::pair<uint64_t, unsigned int> CellFaceKey; // cell_gid + face_id`

Reply:
and later, you actually use `using CellFaceKey = std::pair<uint64_t, unsigned int>;` in cbc_fluds.h

Reply:
I've also reverted this to improve readability.